### DISCUSSION TITLE
Allow mods to enable/disable specific files based on if other mods are active or not

### DISCUSSION THREAD

**Discussion by Jade-Harleyy** [FakeFishGames/Barotrauma#9093] [Ideas and suggestions]
This will allow modders to easily implement additional content that requires other mods, like new items, afflictions, creatures, etc.

**Comment by Ek-Videogames** [comment]
Fun idea and good for compatibility but also prone to exploit problems, interesting concept. 🤔 
Seems like a good workaround for the shortcomings of patches too.
